src/main/scad/puzzlecad/puzzlecad-burr.scad

```diff
@@ -289,6 +289,8 @@ module burr_piece_base(burr_spec, test_poly = undef) {
                 label_scale = is_undef(label_scale_str) ? 0.4 : atof(label_scale_str);
                 assert(label_scale, str("Invalid label_scale: ", label_scale_str));
 
+                label_font = lookup_kv(options, "label_font", default = "Arial");
```

Comment:
Are we certain Arial is the default if no font is specified? (in order to ensure backward-compatibility)

Reply:
https://en.wikibooks.org/wiki/OpenSCAD_User_Manual/Text

Good call, it's actually a Free lookalike that's included with OpenSCAD. I'll update now.

Reply:
Updated to `Liberation Sans`